cpp/src/structs/copying/concatenate.cu

```diff
@@ -53,7 +54,12 @@ std::unique_ptr<column> concatenate(host_span<column_view const> columns,
                    return cudf::detail::concatenate(cols, stream, mr);
                  });
 
-  size_type const total_length = children[0]->size();
+  // get total length from concatenated children; if no child exists, we would compute it
+  auto const total_length = children[0]->size();
```

Comment:
CIs fail: /src/structs/copying/concatenate.cu(60): error: "total_length" has already been declared in the current scope
```suggestion
```